examples/employee-app/src/main/java/com/example/helidon/employee/package-info.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2017-2018, 2019 Oracle and/or its affiliates. All rights reserved.

```

Comment:
This file is left-over from the quickstart example.

Reply:
Removed file. Should be fixed.